models/User.js

```diff
@@ -5,9 +5,9 @@ const userSchema = new Schema({
 	isOwner: Boolean,
 	isPlayer: Boolean,
 	isAdmin: Boolean,
```

Comment:
Looks like this is missing a username/password field. We can add these and use bcrypt to salt the passwords. [http://tphangout.com/how-to-encrypt-passwords-or-other-data-before-saving-it-in-mongodb/](http://tphangout.com/how-to-encrypt-passwords-or-other-data-before-saving-it-in-mongodb/)